tests/magma/compiler/test_compiler.py

```diff
@@ -658,6 +689,36 @@ def test_create_channel_builders_hierarchical_process(self):
         self.assertEqual(chb[0].src_process, p.procs.proc1)
         self.assertEqual(chb[0].dst_process, p.procs.proc2)
 
+    def test_create_channel_builders_ref_ports(self):
+        """Checks creation of channel builders when a process is connected
+        using a RefPort to another process."""
+
+        # create a process with a RefPort (source)
+        src = ProcA()
+
+        # create a process with a var (destination)
```

Comment:
Minor thing: But please check for spelling also in comments to be consistent. The first word of a sentence normally starts with a capital letter. See other comments.